src/theory/fp/theory_fp_rewriter.cpp

```diff
@@ -35,6 +35,7 @@
 
 #include "base/cvc4_assert.h"
 #include "theory/fp/theory_fp_rewriter.h"
+#include "theory/fp/fp_converter.h"
```

Comment:
Order includes alphabetically, this applies to all header files.

Reply:
Will apply globally.